api/src/models/bcmi/annualReport-bcmi.js

```diff
@@ -5,11 +5,11 @@ module.exports = require('../../utils/model-schema-generator')(
   {
     _schemaName:       { type: String, default: 'AnnualReportBCMI', index: true },
     _sourceRefId:     { type: 'ObjectId', default: null, index: true },
+    _master: { type: 'ObjectId', default: null, index: true },
```

Comment:
Alignment of the braces.

Reply:
5d4da993f36eca4bdc1a327ab7784901d7204cc3